src/runtime_src/core/pcie/linux/pcidev.h

```diff
@@ -185,12 +185,12 @@ class dev
 
 private:
   int
-  map_usr_bar();
+  map_usr_bar() const;
 
-  std::mutex m_lock;
-  char *m_user_bar_map = reinterpret_cast<char *>(MAP_FAILED);
+  mutable std::mutex m_lock;
+  mutable char *m_user_bar_map = reinterpret_cast<char *>(MAP_FAILED);
```

Comment:
While mutable mutexes are obvious, m_user_bar_map is less so.  Maybe comment what it is used for and why it is made useable and perceived as constant by const member functions.

Reply:
comment added.